lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -238,7 +256,8 @@ defmodule ExUnit.CLIFormatter do
 
     message =
       "#{test_type_counts}#{config.failure_counter} #{failure_pl}"
-      |> if_true(config.skipped_counter > 0, &(&1 <> ", #{config.skipped_counter} skipped"))
+      |> if_true(config.skipped_counter > 0, &(&1 <> ", " <> skipped("#{config.skipped_counter} skipped", config)))
```

Comment:
I think now we can color the whole thing in yellow? As you originally proposed?

Reply:
Oh, I never really wanted to color the whole thing yellow, just the `1 skipped` part. Do you think the whole thing being yellow is better?

Reply:
I am fine with any approach.